src/devices/icasa.ts

```diff
@@ -63,8 +63,8 @@ const definitions: Definition[] = [
         description: 'Zigbee 3.0 Keypad Pulse 4S',
         meta: {battery: {dontDividePercentage: true}},
         fromZigbee: [fz.command_recall, legacy.fz.scenes_recall_click, fz.command_on, legacy.fz.genOnOff_cmdOn, fz.command_off,
-            legacy.fz.genOnOff_cmdOff, fz.battery, legacy.fz.cmd_move_with_onoff, legacy.fz.cmd_stop_with_onoff],
-        exposes: [e.battery(), e.action(['recall_*', 'on', 'off', 'brightness_move_up', 'brightenss_move_down', 'brightness_stop'])],
+            legacy.fz.genOnOff_cmdOff, fz.battery, legacy.fz.cmd_move_with_onoff, legacy.fz.cmd_stop_with_onoff, fz.command_store],
+        exposes: [e.battery(), e.action(['recall_*', 'on', 'off', 'brightness_move_up', 'brightenss_move_down', 'brightness_stop', 'store_*'])],
```

Comment:
Instead of the wildcard (`*`), can you provide the actual numbers?

Reply:
done :)